src/common/settingsProvider.ts

```diff
@@ -20,6 +39,12 @@ export class SettingsProvider {
     return themeString;
   }
 
+  getNewThemeSettings(): string {
+      const themeSetting = vscode.workspace.getConfiguration().get('workbench.colorTheme');
+      const legacySetting = vscode.workspace.getConfiguration(SETTINGS_STORE_NAME).get('themes');
```

Comment:
What's the difference between these two settings?

Reply:
Good question. We currently have a setting specific to the devtools that just selects between light, dark, and system preference. The new function attempts to auto match the current theme, but then falls back to that setting. I plan on updating the setting to include all the new supported themes (or remove it entirely) once the migration is out in stable edge

Reply:
Tried to make this more clear in updated comments